### ISSUE TITLE
Configurable shortcuts & program into background when copying

### ISSUE THREAD

**Issue by powercurl** [strongbox-password-safe/Strongbox#434]
Is it possible to make the keyboard shortcuts "copy username" and "copy password" configurable? 

In the original Keepass the username is copied with Ctrl+B and the password with Ctrl+C.
This feature would greatly improve the workflow of Strongbox.

Another feature that would improve Strongbox is that when copying e.g. username or password, the program falls into the background and the last opened program comes to the foreground (e.g. browser).

Best regards,
Thomas

**Comment by strongbox-mark** [comment]
Yes, I'll see if that's possible. Perhaps with AutoFill coming on Big Sur this will be less of an issue. Thanks for raising.
